Dockerfile

```diff
@@ -54,13 +54,17 @@ RUN apk add --no-cache make
 WORKDIR /opt/pyroscope
 
 COPY scripts ./scripts
+COPY package.json yarn.lock Makefile ./
+
+# we only need the dependencies required to BUILD the application
+RUN --mount=type=cache,target=/usr/local/share/.cache/yarn/v6 make install-build-web-dependencies
```

Comment:
Didn't know about `--mount=type=cache`. Looks like exactly what we need, awesome find!